code-challenges.js

```diff
@@ -10,36 +10,84 @@
+//     console.log(fruit1)
+// }else if(fruit2.length > fruit1.length){
+//     console.log(fruit2)
+// }
 
 // Pseudo code:
+// determine what data type we are working with
+// set a condition which compares data types which are 2 strings
+// (.length) is used to count the number of index there is in our string
+// we are using "if" statements as our conditional statements to get a bolean equivalent from the 2 items being compared
+// (console.log) is the method we are using to output our statement which is attatched to a data type that is a variable so we can replace true or false with said data type to get the content that is assigned to the variable given to us
+// else if is for the other variable that we have if the 2nd variables index is larger
+
 
 
 // --------------------1) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+ const padresWorldSeriesRuns =(padres1984WorldSeriesRuns.length) + (padres1998WorldSeriesRuns.legnth)
+ console.log(padresWorldSeriesRuns)
+
 // Expected output: 9
 
-// Pseudo code:
+// Pseudo code: will count the amount of index present in the combined array without extracting the actual value.
 
 // --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
 
-const currentCohort = "Golf 2023"
-// Expected output: "3202 floG"
-
+// const currentCohort = "Golf 2023"
+// // Expected output: "3202 floG"
+// const myArray = (currentCohort.split('').reverse().join(""))
+// console.log(myArray)
 // Pseudo code:
+// (myArray)- set our data type
+//(.split) is to split the string into individual strings in order to reverse our array because you cannot reverse a string
+// (.reverse) to reverse our array 
+//(.join) to join all our index in the array to turn into a string
+
 
 // --------------------3) Create the code that will log only the odd numbers from the array using the test variable provided below.
 
-const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
-// Expected output: 13 5 -5 27
+// const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
+// // Expected output: 13 5 -5 27
+// for(let i = 0; i < stockExchange.length; i++){
+//     if(stockExchange[i] % 2 !== 0)
+//     console.log(stockExchange[i])
+// }
 
 // Pseudo code:
+// stockExchange is our given array
+
+// (let i=0) =>starts us off at the 0 index
+// (i < stockExchange.length) => compares each index of the array due to ().length)
+// (i++) => allowing us to go up by 1 after each comparison unstil the end of .length
+// we create a condition using (if) where we can compare data type from the array
+// (stockExchange[i] % 2 !== 0) 
+// [i] => is to compare each value of the array
+// % is to attain the remainder after it is compared to a number which is 2 in our case to equate all even numbers into 0
```

Comment:
- The [i] in itself does not give us the value. When using bracket notation we have to call on the array and the use our brackets to identify the value based on the index. 

- the modulo does give us the remainder, but the remained is not compared to 2. Here we are dividing our current value of the iteration by 2 and evaluating whether the remainder that we receive from that equation is not equal to 0